src/core/Directus/Services/ItemsService.php

```diff
@@ -101,7 +101,9 @@ public function findByIds($collection, $ids, array $params = [], $acl = true)
         $ids = StringUtils::safeCvs($ids, false, false);
 
         try {
-            $this->getAcl()->enforceRead($collection, $statusValue);
+            if (false !== $acl) {
```

Comment:
> (false !== $acl)

This thoroughly annoys me to look at/read! :joy:

Reply:
wrong not is @shealavington (lets run back to front-end land shall we)